packages/flutter_tools/lib/src/base/error_handling_io.dart

```diff
@@ -545,15 +603,18 @@ class ErrorHandlingProcessManager extends ProcessManager {
     Encoding stdoutEncoding = io.systemEncoding,
     Encoding stderrEncoding = io.systemEncoding,
   }) {
-    return _run(() => _delegate.run(
-      command,
-      workingDirectory: workingDirectory,
-      environment: environment,
-      includeParentEnvironment: includeParentEnvironment,
-      runInShell: runInShell,
-      stdoutEncoding: stdoutEncoding,
-      stderrEncoding: stderrEncoding,
-    ), platform: _platform);
+    return _run(() {
+      return _processRun(
+        _getCommandPath(command, workingDirectory),
```

Comment:
Should this do whatever `sanitizeExecutablePath` is doing here: https://github.com/google/process.dart/blob/master/lib/src/interface/local_process_manager.dart#L91 ?

Reply:
This is linked to https://github.com/dart-lang/sdk/issues/37751 which is marked fixed